Create ui-c8y-1021-22-30-clicking-on-simulatorm-alarms-should-not-change-context-8114-graft-release.md (#2914)
SoftwareAG/c8y-docs: content/change-logs/device-management/ui-c8y-1021-22-30-clicking-on-simulatorm-alarms-should-not-change-context-8114-graft-release.md

---
date: ""

> title: Clicking on simulator alarms no longer changes context

Reviewer: ```suggestion
title: Clicking on alarms in the Simulator page no longer redirects to the Alarms page
```

product_area: Device management & connectivity
change_type:
  - value: change-VSkj2iV9m
    label: Fix
component:
  - value: component--KIsStyzM
    label: Device Management app
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: MTM-62195
version: 1021.22.30
---

> Clicking on simulatorm alarms should not change context. (#8114) [GRAFT][release/y2025] (#8204)

Reviewer 2: ```suggestion
In the Cumulocity IoT simulator page, clicking on alarms previously caused the application context to change unexpectedly. Users were redirected to the alarms page instead of showing the alarms in the simulator alarms tab. This behavior has now been corrected. This fix ensures a more consistent user experience within the simulator tool. This change improves the usability and predictability of the simulator for users working with alarms in their simulated IoT scenarios.
```

> In the Cumulocity IoT simulator page, clicking on alarms previously caused the application context to change unexpectedly. Users were redirected to the alarms page instead of showing the alarms in the simulator alarms tab. This behavior has now been corrected. This fix ensures a more consistent user experience within the simulator tool. This change improves the usability and predictability of the simulator for users working with alarms in their simulated IoT scenarios.

Reviewer: ```suggestion
In the **Simulator** page, clicking on alarms previously redirected users to the **Alarms** page instead of showing the alarms in the **Alarms** tab of the **Simulator** page. This behavior has now been corrected. 
```
